### DISCUSSION TITLE
Unlocking Galaxy Brain2

### DISCUSSION THREAD

**Discussion by harshsvaidya** [community/community#130647] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

To unlock the Galaxy Brain badge on GitHub, you need to answer at least 2 discussions. The discussions must be in a public repository, and you cannot mark your own answers to your own questions.

**Comment by harshvaidya04** [comment]
Yes Correct

**Comment by ebndev** [comment]
Hi @harsh432004, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
